packages/client/src/components/notebook/VariablePreview.tsx

```diff
@@ -151,12 +161,16 @@ export const VariablePreview = observer((props: VariablePreviewProps) => {
                     </Typography>
                     <Typography variant="body2">{variable.type}</Typography>
                 </Stack>
-                <Stack direction="row">
-                    <Typography variant="body2" fontWeight="bold">
-                        Value:{' '}
-                    </Typography>
-                    <Typography variant="body2">{val}</Typography>
-                </Stack>
+                {val !== null ? (
+                    <Stack direction="row">
+                        <Typography variant="body2" fontWeight="bold">
+                            Value:{' '}
+                        </Typography>
+                        <Typography variant="body2">{val}</Typography>
+                    </Stack>
+                ) : (
+                    <></>
+                )}
```

Comment:
Can you truncate this with the following? At first pass, I don't believe you need to return a fragment.
    {val !== null && (
                    <Stack direction="row">
                        <Typography variant="body2" fontWeight="bold">
                            Value:{' '}
                        </Typography>
                        <Typography variant="body2">{val}</Typography>
                    </Stack>
                )}